ch_2/repeating_letters/repeating_vowels.py

```diff
@@ -0,0 +1,27 @@
+"""Find the word with the greatest number of repeated vowels."""
+
+from collections import Counter
+
+words = ["ttttttttttest", "program", "iiiiis"]
+vowels = "aeiou"
+
+
+# Option 1
+repeated_vowels = {
+    word: max(Counter(word).items(), key=lambda pair: pair[1]) for word in words
+}
+print(max(repeated_vowels, key=lambda w: repeated_vowels[w][0] in vowels))
+
+
+# Option 2
+def most_repeating_letter_count(word):
+    for letter in word:
+        return Counter(word).most_common()
+
+
+def most_vowels(words):
+    repeats = {word: most_repeating_letter_count(word) for word in words}
+    return max(repeats, key=lambda x: x[0] in vowels)
+
+
```

Comment:
@paulghaddad I was trying to find a way to use itemgetter(), since I used it when I had to find the word with the most repeating letters (see code below). However, I'm having trouble using it when I'm specifically counting the vowels. Thoughts?


```

def most_repeating_letter_count(word):
    return Counter(word).most_common(1)[0][1]

def most_repeating_word(words):
   word_counts = {word : most_repeating_letter_count(word)
                   for word in words}
    return max(word_counts.items(), key=operator.itemgetter(1))[0]
```